kafka/consumer/go.mod

```diff
@@ -8,6 +8,7 @@ module github.com/microsoft/confidential-container-demos/kafka/consumer
 go 1.21
 
 require (
+	github.com/microsoft/confidential-container-demos/kafka/util v0.0.0
```

Comment:
Add a replace rule for this at the bottom of the file.  Should be `replace github.com/microsoft/confidential-container-demos/kafka/util => ../util` https://go.dev/doc/modules/gomod-ref#replace